### DISCUSSION TITLE
Alert Dashboard Layout

### DISCUSSION THREAD

**Discussion by ryanbuening** [datalust/seq-tickets#1470] [Ideas]
Depending on your browser window size, it currently is not easy to see the full alert title when viewing the Alerts dashboard:

![image](https://user-images.githubusercontent.com/812974/152384037-7de6c8d2-55e9-4a36-8b4b-60411060bf0c.png)

Would be nice to have a "compact" view which could show the full title.

Would also be nice to have the ability to group alerts. Currently, I am prefixing my alert titles with a name and a pipe | to provide some ad hoc grouping.

**Comment by liammclennan** [comment]
Thanks for these ideas too Ryan. You can see the full alert title in the tooltips if you place your mouse over the title.
